config/package/socorro/alembic.ini

```diff
@@ -11,7 +11,7 @@ file_template = %%(rev)s_%%(slug)s
 # the 'revision' command, regardless of autogenerate
 # revision_environment = false
 
-sqlalchemy.url = postgresql://postgres@localhost/breakpad
+sqlalchemy.url = postgresql://breakpad_rw:aPassword@localhost/breakpad
```

Comment:
I'd like to make sure this doesn't break the RPM (travis doesn't test this but we have a jenkins job that builds it)